### PR TITLE
serialization corrections

### PR DESCRIPTION
**Pull request by elisarver**
- working_dir to dir
- output from string to []byte
- coverage for marshal/unmarshal

### CODE REVIEW

**Comment by elisarver** [comment]
I'm always forgetting to update the README.md

File: capture.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ type Capture struct {
 	Env []string `json:"env,omitempty"`
 
 	// Output provides the combined output from the name as a string.
```
Comment (shairozan) [inline capture.go]: Trivial, but we may want to update the comment to reflect it's a []byte now or just remove the comment?

File: capture_test.go [hunk 2]
```diff
@@ -145,23 +156,108 @@ func TestRerun(t *testing.T) {
...
+
+func AllMatcher(t *testing.T, want, got command.Capture) {
+	ExitCodeMatcher(t, want, got)
+	OutputMatcher(t, want, got)
+	ArgsMatcher(t, want, got)
+	NameMatcher(t, want, got)
+	DirMatcher(t, want, got)
+}
+
+func OutputMatcher(t *testing.T, want, got command.Capture) {
```
Comment (shairozan) [inline capture_test.go:195]: This is something I need to start working on. Tests for me are generally a "Just get it done" and move along, so I've never gone back to evaluate if the tests themselves could be made more efficient / clean

**Review by shairozan — APPROVED** [review]
Looks good. I like the structure of the tests since it's basically similar to how a lot of the BDD suites I've worked with are. Just cleaner in the end since it doesn't have the extra weight

**Pull request merged**